setup.cfg

```diff
@@ -29,3 +32,49 @@ addopts =
     # Previous versions included the following, but it's a bad idea because it
     # hard-codes the value and makes it hard to change from the command-line
     tests/
+
+[metadata]
+name = pycobertura
+version = 2.1.0
+description = "A Cobertura coverage parser that can diff reports and show coverage progress."
+author = "Alex Conrad"
+author_email = "alexandre.conrad@gmail.com"
+long_description = long_description
+long_description_content_type = 'text/markdown'
+maintainer = "Alex Conrad"
+maintainer_email = "alexandre.conrad@gmail.com"
+license_file = LICENSE
+keywords = "cobertura coverage diff report parser parse xml"
+url = "https://github.com/aconrad/pycobertura"
+classifiers =
+    Topic :: Software Development :: Libraries :: Python Modules
+    License :: OSI Approved :: MIT License
+    Natural Language :: English
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
```

Comment:
Can we remove 3.10 until we have a working Travis build?

Reply:
yes

Reply:
yes